force_wfmanager/central_pane/plot.py

```diff
@@ -36,12 +36,20 @@ class Plot(HasStrictTraits):
     #: Datasource of the plot (used for selection handling)
     _plot_index_datasource = Instance(ArrayDataSource)
 
+    #: Button to reset plot view
+    reset_plot = Button('Reset View')
+
+    reset_enabled = Property(Bool(False), depends_on="_plot_data")
```

Comment:
Passing False here does not help. A property is always invoked to determine its value.